src/test/kotlin/com/exactpro/th2/codec/fixng/FixNgCodecTest.kt

```diff
@@ -41,8 +41,19 @@ import java.math.BigDecimal
 import java.nio.charset.StandardCharsets
 import java.time.Instant
 import java.time.LocalDateTime
+import kotlin.collections.set
 import kotlin.math.max
 
+/*
+ * 8:
+ *  - possible values (low because conn should check it)
+ * 9:
+ *  - should cover message from current position to 10 tag
+ * 10:
+ *  - must have value length 3
+ *  - should be coder length
+ */
```

Comment:
note: I am not sure that this is the right place for a comment like this. We probably should spread the comment across individual tests that actually test those tags